### PR TITLE
Health check

### PR DESCRIPTION
**Pull request by txgruppi**
Added a simple `/health-check` endpoint.

I'm not sure I did right with the API version. Should I update the `@apiVersion` in all files?

### CODE REVIEW

**Comment by txgruppi** [comment]
`rebase master`

**Comment by pnagurny** [comment]
Thanks for another PR @txgruppi . The functionality looks good. The idea is to eventually be able to support multiple API versions at the same time. As long as we don't have any breaking changes, we can keep it simple and just serve the latest and greatest. Eventually if we do a new MAJOR version with breaking changes, we'll want to redesign things.

For now what I do is every time I change the API, I search and replace every instance of the old version and replace it with the new version.

In your case you will want to replace every instance of "1.0.1" and replace it with "1.1.0". The occurrences should be:

1. core/api*.go API doc comments (the documentation is generated from these)
2. core/api/version.go
3. core/ws/ws.go

The oa-web client is set to accept ^1.0.1 which means that it will automatically accept any new patch or minor level changes.

**Comment by txgruppi** [comment]
@pnagurny, I update the files with the new version.

Thanks for explaining and for project.

Cheers.

File: core/ws/ws.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 	"sync"
 )
 
-const version = "1.0.1"
+const version = "1.1.1"
```
Comment (pnagurny) [inline core/ws/ws.go]: should be 1.1.0

**Comment by pnagurny** [comment]
Besides the one typo above ^^ everything else looks good. Thanks!

**Pull request merged**